aws/awserr/types_1_13.go

```diff
@@ -0,0 +1,31 @@
+//go:build go1.13
+// +build go1.13
+
+package awserr
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Is reports whether this error matches target.
+func (b baseError) Is(target error) bool {
+	for _, e := range b.errs {
+		if errors.Is(e, target) {
+			return true
+		}
+	}
+	return false
+}
+
+// As finds the first error in errs that matches target, and if so,
+// sets target to that error value and returns true. Otherwise, it returns false.
+func (b baseError) As(target interface{}) bool {
+	for _, e := range b.errs {
+		fmt.Printf("%T, %T\n", e, target)
```

Comment:
Looks like a leftover debugging statement?

Reply:
Thanks! I've removed it.